### PR TITLE
updated groupId query, fixed auth issue for core dashboard

### PR DESCRIPTION
**Pull request by Oluwadaminiola**

### CODE REVIEW

File: web/ui/dashboard-component/projects/convoy-dashboard/src/lib/convoy-dashboard.service.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { HTTP_RESPONSE } from './models/http.model';
 })
 export class ConvoyDashboardService {
 	token: string = '';
-	authType: 'Bearer' | 'Basic' = 'Basic';
+	authType!: 'Bearer' | 'Basic';
```
Comment (horlah) [inline web/ui/dashboard-component/projects/convoy-dashboard/src/lib/convoy-dashboard.service.ts:10]: Please why did you change to this @Oluwadaminiola

Comment (Oluwadaminiola) [reply web/ui/dashboard-component/projects/convoy-dashboard/src/lib/convoy-dashboard.service.ts:10]: this was causing an error on SAAS as authType kept returning BASIC, so I decided to rely on the values coming from the component itself

**Review by horlah — APPROVED** [review]
